### DISCUSSION TITLE
Error 404 Pages

### DISCUSSION THREAD

**Discussion by ghost** [community/community#47808] [Other Feature Feedback, Questions, & Ideas]
When I enter: http://jashanjudge87.github.io
I get: 404 There isn't a GitHub Pages site here.

I deleted the repository and made a new one after it had not been working for over 48 hours. I'm still having the same problem. I have also cleared the cache and tried it on an incognito tab, different browsers no change. Could anyone help me out?

Repository: https://github.com/jashan87/jashanjudge87.github.io

Settings image: <img width="799" alt="Screenshot 2023-02-17 at 1 40 37 PM" src="https://user-images.githubusercontent.com/123149482/219756938-f12aed87-5854-4c1f-b35b-cc6003be896c.png">

Error image: <img width="805" alt="Screenshot 2023-02-17 at 1 41 27 PM" src="https://user-images.githubusercontent.com/123149482/219757109-f595fd3c-ca79-4e67-90b2-6d2c1abfaa96.png">

**Comment by akifsohee** [comment]
solved?

**Reply by ghost** [reply]
why do you say that? it still does not work

**Comment by shree675** [comment]
@jashan87 Try this link: https://jashan87.github.io, it works.
You must have changed the repository name to 'jashan87.github.io', hence the new repository URL is https://github.com/jashan87/jashan87.github.io.
